harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/localfilesystem/LocalFsHarvesterFileVisitor.java

```diff
@@ -230,6 +230,15 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
         return FileVisitResult.CONTINUE;
     }
 
+    /** Reset the milliseconds of a date */
+	private Date resetMilliseconds(Date recordDate) {
```

Comment:
Maybe can be simplified using this:

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/time/DateUtils.html#truncate(java.util.Date, int)